front/src/applications/operationalStudies/components/SimulationResults/SpeedSpaceChart/SpeedSpaceChart.jsx

```diff
@@ -57,6 +61,8 @@ export default function SpeedSpaceChart(props) {
   const [rotate, setRotate] = useState(false);
   const [showSettings, setShowSettings] = useState(false);
 
+  const [restrictionPower, setRestrictionPower] = useState('');
```

Comment:
Can you move it above ? (alphabetical order for the state vars)

Reply:
done